src/registry_schemas/schemas/special_resolution.json

```diff
@@ -32,6 +32,9 @@
                 },
                 "signatory": {
                     "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/person"
+                },
+                "resolutionConfirmed": {
```

Comment:
This is required for the UI to save the checkbox for the draft: 

![image](https://user-images.githubusercontent.com/3484109/186968768-b0a64570-ace3-475b-9b2b-40ea780896fd.png)

Reply:
bcgov/entity/13289